src/Views/AddDog/AddDog.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import DogForm from '../../Comps/DogForm';
+import { addDog } from '../../services/fetchdogs';
+
+
+export default function AddDog() {
+
+  const [img, setImg] = useState('');
+  const [Name, setName] = useState('');
+  const [Age, setAge] = useState('');
+  const [Breed, setBreed] = useState('');
+  const [Desc, setDesc] = useState('');
+  const [error, setError] = useState('');
+  
+  const history = useHistory();
+  
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await addDog({ Name, Age, Desc, Breed, img });
+      history.push('/');
+    } catch (e) {
+      setError('Make sure your fields are filled in.');
+    }
```

Comment:
Be sure to add in a success message, perhaps before the user is redirected back to home. Also, missing a NavLink to the AddDog component within a header or NavBar.